tutorials/hsdk-ngc-container/README.md

```diff
@@ -0,0 +1,97 @@
+# Holoscan SDK VS Code DevContainer Template
+
+This directory contains the necessary files and configurations to start developing Holoscan applications in the [Holoscan NGC Container](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/containers/holoscan) using a [Dev Container](https://code.visualstudio.com/docs/devcontainers/containers) for [VS Code](https://code.visualstudio.com/).
+
+## Prerequisites
+
+- [Docker](https://www.docker.com/)
+- [VS Code](https://code.visualstudio.com/) with the [Dev Container Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+  - Install [Dev Container Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) via command line
+    ```bash
+    code --install-extension ms-vscode-remote.remote-containers
+    ```
+- [NVIDIA Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html)
+- [NVIDIA CUDA Toolkit](https://developer.nvidia.com/cuda-downloads)
+- [Holoscan SDK container 2.3 or later](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/containers/holoscan)
+
+## DevContainer Setup
+
+1. Download everything in this directory to a folder of your choice.
+2. Open [.devcontainer/devcontainer.json](.devcontainer/devcontainer.json#L10), find and change the value of `HOLOSCAN_SDK_IMAGE` to a version of [Holoscan SDK container](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/containers/holoscan) image you want to use.
+  :bulb: **Important**: Holoscan SDK container v2.3 or later is required.
```

Comment:
Looks like `:bulb:` is not displaying properly in markdown

Reply:
Ok. I need to switch to a different md renderer in VS code :joy: